src/libslic3r/PrintObject.cpp

```diff
@@ -428,6 +432,25 @@ void PrintObject::generate_support_material()
     }
 }
 
+void PrintObject::prepare_adaptive_infill_data()
+{
+    float fill_density = this->print()->full_print_config().opt_float("fill_density");
```

Comment:
full_print_config() is a DynamicPrintConfig: A hash table.
Please use Print::config() or PrintObject::config()